lib/workos/version.rb

```diff
@@ -2,5 +2,5 @@
 # typed: strong
 
 module WorkOS
-  VERSION = '0.8.1'
+  VERSION = '0.9.0'
```

Comment:
I think we should be good for this to be 0.8.2 as it's not a breaking change of any kind

Reply:
@willmanduffy Hm I do think this would be minor bump instead of a patch based on our [versioning policy](https://github.com/workos-inc/workos/wiki/SDK-Release-Process#versioning-policy). Since you're right, it doesn't include breaking changes, and also isn’t a bug fix 🤔

Reply:
Yeah, I think that's totally right if we're past 1.0.0. I personally think that versioning policy is a bit aggressive for the amount of change we're doing right now. Semver points out that pre-1.0.0 can be the wild west

> Major version zero (0.y.z) is for initial development. Anything MAY change at any time. The public API SHOULD NOT be considered stable.

https://semver.org/#spec-item-4

That said if all the others are merged as minor bumps right now we should go ahead with that for consistency

Reply:
Ah, got it! Thank you, I appreciate the spec pointer.
I'm on board with your proposal: we can merge this one as a minor bump for consistency. (And I'll be more conservative with versioning moving forward as well.)